aunt-leahs-app/src/constants.jsx

```diff
@@ -0,0 +1,18 @@
+/* new constant should be child of 'constants'*/
+const constants = {
+    headerConstants:{
+        headerText:{
+            headOne:"Aunt Leah's Place",
```

Comment:
Since no other component should ever use these, I think these constants should be in the header component. And since they're static, they should be all caps (imo).

Also, maybe consider naming them something else, like, "MAIN_HEADER" and "SUB_HEADER"

Reply:
@codygagnon suggested putting all the constants all together in one file. Hence, I've moved it out of the header component into /src/components.jsx. Noted on the renaming.